proto/c1/connector/v2/event_feed.proto

```diff
@@ -41,6 +49,29 @@ message ListEventsResponse {
   repeated google.protobuf.Any annotations = 4;
 }
 
+message ListEventFeedsRequest {
+  uint32 page_size = 1 [(validate.rules).uint32 = {
+    ignore_empty: true
+    lte: 250
+  }];
+  string page_token = 2 [(validate.rules).string = {
+    ignore_empty: true
+    min_bytes: 1
+    max_bytes: 1048576 // 1MB
+  }];
+  repeated google.protobuf.Any annotations = 3;
+}
+
+message ListEventFeedsResponse {
+  repeated EventFeedMetadata list = 1;
```

Comment:
Connectorbuilder's ListEventFeeds doesn't obey the page token or page size right now, so I think we can kill the page size/token and just always list all event feeds. Although we have no limit on the number of feeds, if a connector has more than a few then we are doing something wrong.